src/main/java/com/myThreadPool/MyQueue.java

```diff
@@ -0,0 +1,54 @@
+package com.myThreadPool;
+
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class MyQueue<T> {
+
+    private final Queue<T> queue = new LinkedList<>();
+
+    private final int EMPTY = 0;
+    private int MAX_TASK_IN_QUEUE = -1;
```

Comment:
`MAX_TASK_IN_QUEUE` can be `final` (you will have to remove the ` = -1` initialisation)